community/py-greenlet/APKBUILD

```diff
@@ -11,8 +11,7 @@ license="MIT"
 depends=""
 makedepends="python2-dev py-setuptools python3-dev"
 subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
-	ppc64le_clobbering.patch"
```

Comment:
@J0WI did you test ppc64le build? or it was fixed upstream?

Reply:
I think this has been addressed in https://github.com/python-greenlet/greenlet/commit/9c8a65fddcffe161f38679a0b3289de9b25e2bc6